src/routes/environment/find.ts

```diff
@@ -34,6 +35,9 @@ const controllerGeneratorOptions: ControllerGeneratorOptionsWithoutClientOrSuppl
       environment: environmentSchema.required(),
       environmentRights: Joi.array().items(Joi.string()).required().example(['STATIC', 'USERS'])
         .description('See the getting started section about rights'),
+      notificationLevel: Joi.number().allow(null).required()
```

Comment:
This should be an exception to my remark that "everywhere where `environmentRights` are supplied in a response, also supply `notificationLevel: 0 | 1 | 2 | null`". The `environmentRights` here are not about the user performing the request, but the user of the row and I don't think it makes sense that the requesting user knows about that. So can you drop `notificationLevel` again here?